app/src/main/java/org/mozilla/tv/firefox/FocusLossFrameLayout.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import androidx.recyclerview.widget.RecyclerView
+
+/**
```

Comment:
Can we also add a comment to `default_channel.xml`'s RecyclerView referencing this file and this fix? I'm concerned that this fix is hard to find because it's on each item, rather than on the parent. For example, if someone tries to implement a new channel, they make look at the `default_channel` layout but not each item and may run into the same issue.